internal/batches/service/service.go

```diff
@@ -645,23 +644,11 @@ func (svc *Service) FindDirectoriesInRepos(ctx context.Context, fileName string,
 	}
 
 	findInBatch := func(batch []*graphql.Repository, results map[*graphql.Repository][]string) error {
-		const searchQueryTmpl = `%s: search(query: %q, version: V2) {
-        results {
-            results {
-                __typename
-                ... on FileMatch {
-                    file { path }
-                }
-            }
-        }
-	}
-`
-
 		var a strings.Builder
-		a.WriteString("query DirectoriesContainingFile() {\n")
+		a.WriteString("query DirectoriesContainingFile {\n")
```

Comment:
that's illegal in graphql, but our library has an open ticket for it. It might break in the future, though.